src/main/scala/cromwell/engine/WorkflowActor.scala

```diff
@@ -63,68 +65,75 @@ class WorkflowActor private(binding: WdlBinding, actualInputs: Map[FullyQualifie
   private def notStarted: Receive = {
     LoggingReceive {
 
-      case Start(backend) =>
+      case Start(system, backend) =>
         _primeMover = Option(sender())
-        maybeExecutionStatusStore = Option(new ExecutionStatusStore(binding))
+        maybeExecutionStore = Option(new ExecutionStore(binding))
         maybeSymbolStore = Option(new SymbolStore(binding, actualInputs))
         maybeBackend = Option(backend)
-        executionStatusStore.runnableCalls.foreach { call =>
-          startCallActor(symbolStore, call)
-        }
-        primeMover ! Started
+        // Poll for runnable calls.
+        system.scheduler.schedule(
```

Comment:
Could this lead to the system getting backed up? What if for some reason the previous call hadn't completed? Would it be better to have the thing checking for runnable calls send the actor the next CheckExecutionStatus message to ensure the previous one was complete?

Actually, what's the prevent some rogue actor from sending these messages in?

Wouldn't the behavior of CheckExecutionStatus only be necessary initially and when a call is done? Perhaps just have a function which both call (and that function would call startCallActor and such)?